src/main/java/org/takes/facets/slf4j/TsLogged.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package org.takes.facets.slf4j;
+
+import java.io.IOException;
+import org.takes.Request;
+import org.takes.Take;
+import org.takes.Takes;
+import org.takes.tk.TkEmpty;
+
+/**
+ * Logs Takes.route() calls.
+ *
+ * <p>The class is immutable and thread-safe.
+ * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
+ * @version $Id$
+ * @todo #28:30min/DEV This class not implemented yet, but has to be.
+ *  Please implement it like TkLogged and don't forget about unit tests.
+ *  In addition, need to be implement classes BkLogged, FtLogged, PsLogged
+ *  which will make loggable interfaces Back, Front, Pass. They should all be
+ *  implemented also like TkLogged.
+ * @since 0.11.2
+ */
+public final class TsLogged implements Takes {
+    @Override
+    public Take route(final Request request) throws IOException {
+        return new TkEmpty();
```

Comment:
it's better to throw `UnsupportedOperationException` when something is not implemented yet